canonical/ubuntu.com
# Build an example case study using macros

## Done

- Build an example case study using macros
- Figma design of example case study - [link](https://www.figma.com/design/AeIJ3GsqnJCHtiBYhkTuTV/24.10-Case-study-template?node-id=57-1642&t=sUGdvJIWpE4J9LMw-0)

## QA

- Go to https://ubuntu-com-14160.demos.haus/case-studies/example
- Check that the example case study matches the Figma design 25/75

## Issue / Card

Fixes [WD-14051](https://warthogs.atlassian.net/browse/WD-14051)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-14051]: https://warthogs.atlassian.net/browse/WD-14051?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
